File: app/src/main/java/cn/intret/app/picgo/model/event/BaseMessage.java
package cn.intret.app.picgo.model.event;


public class BaseMessage {
    String mDesc;

    public BaseMessage() {
    }

    public BaseMessage(String desc) {
        mDesc = desc;
    }

    public String getDesc() {
        return mDesc;
    }

    public BaseMessage setDesc(String desc) {
        mDesc = desc;
        return this;
    }

    @Override
    public String toString() {
        return "BaseMessage{" +
                "mDesc='" + mDesc + '\'' +
                '}';
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/BaseMessage.kt
package cn.intret.app.picgo.model.event


open class BaseMessage {
    internal var mDesc: String? = null

    constructor()

    constructor(desc: String?) {
        mDesc = desc
    }

    fun getDesc(): String? {
        return mDesc
    }

    fun setDesc(desc: String?): BaseMessage {
        mDesc = desc
        return this
    }

    override fun toString(): String {
        return "BaseMessage{" +
                "mDesc='" + mDesc + '\''.toString() +
                '}'.toString()
    }
}
